crates/c-api/CMakeLists.txt

```diff
@@ -1,140 +1,174 @@
 cmake_minimum_required(VERSION 3.10)
 project(wasmtime C)
 
-set(WASMTIME_USER_CARGO_BUILD_OPTIONS "" CACHE STRING "Additional cargo flags (such as --features) to apply to the build command")
+option(WASMTIME_USER_CARGO_BUILD_OPTIONS "Additional cargo flags (such as --features) to apply to the build command" OFF)
```

Comment:
Sus. `option` is for cache variables that are booleans, but it looks like `WASMTIME_USER_CARGO_BUILD_OPTIONS` is supposed to be a string. This doesn't look correct.

Reply:
Thanks! I didn't really realize that `set(...)` here was doing that. I'll switch back.